repose-aggregator/components/filters/rate-limiting-filter/src/main/java/org/openrepose/filters/ratelimiting/RateLimitingFilter.java

```diff
@@ -195,10 +195,10 @@ public boolean isInitialized() {
     }
 
     private static class RateLimitingConfig {
-        public RateLimitingService rateLimitingService;
-        public Optional<Pattern> describeLimitsUriRegex;
-        public boolean includeAbsoluteLimits;
-        public boolean isOverLimit429ResponseCode;
-        public int datastoreWarnLimit;
+        private RateLimitingService rateLimitingService;
```

Comment:
The parent of an inner class has access to it's private members? That does not make sense to me.